### PR TITLE
Transcriptomics: Add step to annotate genes with name, location from GTF

### PR DESCRIPTION
**Pull request by bebatut**
using `Annotate DESeq2/DEXSeq output tables` tool directly to DESeq2 output

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -111,6 +111,7 @@ icon-tag:
   galaxy-dropdown: fa-caret-down
   galaxy-wf-connection: fa-arrows-h
   galaxy-wf-new: fa-plus
+  galaxy-scratchbook: fa-th
```
Comment (shiltemann) [inline _config.yml]: we already have this icon 4 lines above :)

Comment (erasche) [reply _config.yml]:
```suggestion
```

**Comment by bebatut** [comment]
@galaxyproject/training-transcriptomics any feedback on that PR?

**Comment by bebatut** [comment]
This PR is out of date and most of it has been fixed with #1549

**Pull request closed without merging**